content/influxdb/v2.0/reference/cli/influx/write/_index.md

```diff
@@ -47,6 +47,7 @@ influx write [command]
 | `-o` | `--org`            | Organization name                                                     | string     | `INFLUX_ORG`          |
 |      | `--org-id`         | Organization ID                                                       | string     | `INFLUX_ORG_ID`       |
 | `-p` | `--precision`      | Precision of the timestamps (default `ns`)                            | string     | `INFLUX_PRECISION`    |
+|      | `--rate-limit`     | Throttle write rate (examples: `5 MB / 5 min` or `1MB/s`).            | string     |                       |
```

Comment:
Seems like there is a bit of an interesting syntax here. In the future we might want to describe the syntax in detail and provide further examples.